monkey-runner/build.gradle

```diff
@@ -0,0 +1,17 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.1.2'
```

Comment:
pulling put the build script dependencies - https://github.com/novoda/spikes/pull/144/commits/109e83666a9ea1864b5c3813634fa7410ac2fa76

pulling out the compile dependencies - https://github.com/novoda/spikes/pull/144/commits/722d9693d24b9d9a1edc2b205c95386caebeb170